ext/LegendSpecFitsRecipesBaseExt.jl

```diff
@@ -921,4 +922,124 @@ end
         report.x, report.gof.residuals_norm
     end
 end
+
+@recipe function f(report::NamedTuple{(:h_before, :h_after_low, :h_after_ds, :dep_h_before, :dep_h_after_low, :dep_h_after_ds, :sf, :n0, :lowcut, :highcut, :e_unit, :bin_width)})
+    legend := :topright
+    foreground_color_legend := :silver
+    background_color_legend := :white
+    size := (1000, 600)
+    xlabel := "Energy ($(report.e_unit))"
+    ylabel := "Counts / $(round_wo_units(report.bin_width, digits=2))"
+    framestyle := :box
+    thickness_scaling := 1.2
+    xticks := (0:300:3000, ["$i" for i in 0:300:3000])
```

Comment:
why is this needed?
Looks to me like this could just be `0:300:3000`.

Reply:
It is needed because CoPilot suggested it